src/index.html

```diff
@@ -19,6 +19,7 @@
H0+tmF3SdiB26geTHf1eqFewhXeakpPx3eSRdoxjRWIRS+fmfB0m4h/kfuH/APar/wCFtu3z/on6eR6Z0j4m7YRHyjcf0csPY9quOfP+h08zc0vF+JW+8a1ru4QNcLctftDT9FVns+1ziPD/AHY/4s3VSfP89SVHxbXcw0//ABnD9KkqJ2Fv+S/6GyqT/F/ZuuG+JJJpOrlbqQSC2KZo0/EXZmgW8R9VUurONOGuD9V/RvCo28MtC52cEpuqeTM0Fe5tK/TUYz6/nzOdOOmTRkVg1FkAsgCAIAgCAIAgCAIAgPDomndoPooZ21GbzOCb7UjZSa4M8Gmj/CFC7C2f/rXkZ6SfWc+4g6S8Pp5HxMhkmcwlpLSGMzA2IzE3Oo3AstHsu1fwer+5npp9ZUazpWqHm0FNCz8xkld9C39Fp/iLT+L82Z6efWQv+oseqLmNs9huIqfQeuQn6raGy7WPwZ72zDrTfM8OwfH5vaFVr+OXq/o54/RXqdKMI6YLC7CpVq0VL9TGe7J9/wDDjEpAC/qrnk+Qkjzs0j5EqTSyL/XUY8M+CJtN0VVJtnnib35Q93yvlus6CN7QhyTNrB0Uw+/UyH8rWN/W6zoIntGXKKJUXRdRC+aSd3xMH6MTQjR7QqdSJcfRvhoOrJHbaGR4/lIOqaUau+rda8iVB0d4YT2aUnzknI+r7I1FG0Li5qbov0X2JMHR7QM2o2n8znP/AJnFcm6ubvhRpeLafpn79x1KNCT31angt3qSxwhRj/0MGn+VH/ZcWS2lnLc/Bv6HQXRLqJDMEgbtTRjyjaP6KtKF5zU/U3Th2B+EwHenj9Y2/wBlo3cR/l6mcQ7Dy7C6fS8EWm3YZp5aKPpaq+J+bM6YmeGFjBZrWtHc0AD5BRym5PMmZwiLhNf18ZcW5S18sZadbOje5m/iBfycFLWpdFLGc5Sfms/0YjLKJpUJsMo7lnIPqwCdhkm7fUf1XoNiV/epPvX1+hVuI/EbBehKwQBAEAQBAEAQBAEBjlma32iAsqLfBG0YylwR4p6kPJsDbvI0K2lBx4m06bhxZnWhGfCEBpP+j8N6wyGjgLzrcsaRfvsdPogNrT0UUf8A5cbGfla1v6BAZ0BqsR9v0CkjwONffu+C+pGWxSNfiGNU0Dg2WVrHEXAO9ts1hsPE6JlEkKU5rMUSqSrjlYHxPa9jtnNIc0+RGiGsouLw1glRQOdsPXksNklKhUqe6vEmw0DR7Wv6LRyZ0aVhCO+e/wCRMa0DYLUvKKSwj6hkIAgCA+WTAPhYO4LR04PikZyyuy8IsdLK/wC01DI5n9Y6GNzWNz5GMJztb1guGA2DgL371rKhSlxivJDU+s1mC4vT0rZKSXrZJYJpGNa2OaaUxOPWRElocSMkgbmcd2nVRSsreTy4LyNlUkuZaqSNkjGv6tzMwvlfo4eDgCbH1Ub2bav4F6meln1mQ0TO76lRvZNq/h9X9zPTT6z7HSNabi9x4rajs2hRmpwymu0xKrKSwyQugRhAEAQBAEAQBAeJX2BNibchuspZeDMVl4Iv3z+5g+ZUn6ce0m/Sh2v0PcVCwantHvdqsOrJ7jWVaT3Ld3EpRkQQHwlAVuTG5qpxZh4ZkabPq5A50QI3bE1paZnciQQ0a6kiyAm8PYlLJ1sU7WiaB4Y8svkeHNa9sjASS0FrvZJJBB1OhIG4QGqxH2/QKSPA419+74L6kZbFIpmAzyQukq5szo53GOdwaSaaWJ72hrmjtCHKQL8rZjo640ydSVs6kVFbscO1NfPJuMtJFVU01JNE/wC0ydXLHG5jmytLHOEwa06PZlF3c2kg8ra5LVO2iopS9rHWXgBYLB9QBAEB8ugPE7C5pAcWkggOFri4tcX0uPFAaXh/EpXiSnnLRVQWa820e036uoa38L7ajk4OHJASOGKesjgDa2Zk02Z3bYMoy37I2Gvp890BtkAQBAEAQBAEAQBAEAQBAEAQBAEAQBARMVxKKmidLK7K1tuRJJJsGtaNXOJNgBqUBWKAOxZnWyuDKMkgUzXdt5abEVTh7Oo1hHxE7ID1V4jJUDqsOAbHH2HVNvumgaFkDRYSvG1x2G95IyrnbQpLT02/2eSeM9/YvPBLSe/SSMHw1sALIy8ue7M97nFz3vIAL3u5mwA7gAALALzruLi4qKKk8vckm0kWtEYIssTLABeuo0+jgoZzjmyjJ5eSLU0he+97CwU6lhHPr2sqtTOcIzQ0rG8te8rDk2T0rWnT3pb+tmrxDALymenmdTzOtnc0NcySwsOtido4gbOBDtLXtosFgwYFwyIZ3VMzo5J3NyZo4Wwta29zZgc4lxNruLidBsEBYkAQBAYKqoDB4nYLKWSvcV1SjnnyNY2ocHZr6/8ANFJjcceNxUjPpM7za08wcLhRtYO3RqqrHUjT8S4bISyppx/iYM2UbdbE63WU7j3OsCDyc1p774JTY4TiUdTCyaI3Y8XFxYg7FrhycCCCORBQExAEAQBAEAQBAEAQBAEAQBAEAQBAEAQGs4hwx1REGxvDJGPjljcRcB8bg4Bw5tOrTzsTbVAV6m4WqJ5ppKwsiim6syUsD3OZK9jcpfLIWscWkWBYBqGtzE7IC4RQta0Na0NaAAGgAAAaAADYLDSe5gw01JlcT8vJcyz2crerKf8A89i/N3cTTq6kkSl1CEIAgCAIAgCAIDxLEHCxWU8EdSnGpHTI1zaF2ax27+//AHW+rccuNjPpNL4dZsmMAFgozrRiorC4HpDYhUGGxwvldGCOuf1jm3OXPYNLmt90mwJtuddyUBNQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEAQBAEB//2Q==" />
```

Comment:
why are you using blob for image?

Reply:
Please have a look when you are free! Thank you!